query/src/test/java/org/infinispan/query/blackbox/LocalCachePerfProgrammaticConfTest.java

```diff
@@ -32,16 +32,17 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
             .enable()
             .addIndexedEntity(Person.class)
             .addIndexedEntity(AnotherGrassEater.class)
-            .addProperty("default.directory_provider", "local-heap")
-            .addProperty("default.chunk_size", "128000")
-            .addProperty("default.indexmanager", "near-real-time")
-            .addProperty("error_handler", StaticTestingErrorHandler.class.getName())
-            .addProperty("default.indexBase", indexDirectory)
-            .addProperty("default.indexwriter.merge_factor", "30")
-            .addProperty("default.indexwriter.merge_max_size", "4096")
-            .addProperty("default.indexwriter.ram_buffer_size", "220")
-            .addProperty("default.sharding_strategy.nbr_of_shards", "6")
```

Comment:
Are lucene low level properties still supported?

Reply:
[HSEARCH-3776](https://hibernate.atlassian.net/browse/HSEARCH-3776) will be present in the next release that will be the Beta7, that we will release soon, together with [HSEARCH-3777](https://hibernate.atlassian.net/browse/HSEARCH-3777) for similarity.